modules/tinymce/src/themes/silver/test/ts/browser/window/SilverDialogAriaLabelTest.ts

```diff
@@ -52,7 +53,7 @@ UnitTest.asynctest('WindowManager:inline-dialog Test', (success, failure) => {
         '.tox-dialog { background: white; border: 2px solid black; padding: 1em; margin: 1em; }'
       ]),
       sTestDialogLabelled({ inline: 'toolbar' }),
-      sTestDialogLabelled({ inline: 'not-inline!!' }),
+      sTestDialogLabelled({}),
```

Comment:
I've seen this written two different ways (with and without a space) in here

```suggestion
      sTestDialogLabelled({ }),
```